content/en/monitors/manage/status.md

```diff
@@ -178,8 +178,9 @@ You can obtain a JSON export of any monitor from the monitor's status page. Clic
 [8]: /notebooks
 [9]: /monitors/configuration/?tab=thresholdalert#evaluation-window
 [10]: https://app.datadoghq.com/event/explorer
-[11]: /events/
-[12]: https://app.datadoghq.com/monitors#create/import
-[13]: /monitors/configuration/?tab=thresholdalert#notification-aggregation
+[11]: /account_management/audit_trail/
+[12]: /blog/audit-trail-best-practices/
```

Comment:
```suggestion
[12]: https://www.datadoghq.com/blog/audit-trail-best-practices/
```
One last fix.

Also, after I approve, should I merge this PR?

Reply:
Hi @urseberry,
we can merge it tomorrow when the change will be available in all the DC.

Thanks 🙇‍♂️